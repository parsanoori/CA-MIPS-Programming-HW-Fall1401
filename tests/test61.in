84
1050300208
690542678
174208411
763140492
889732650
141373030
507498319
394198436
377202253
698664947
768377218
9446006
433883883
180003713
1056134075
508218917
157742511
48538818
62017944
688713923
2022756
834508085
388111810
209245850
742862699
692669654
1012674018
982311815
721483381
250519016
293993014
38106522
41096124
361996174
734929698
24023255
468642530
17406770
473856419
172892190
784394742
878708369
557379720
526105145
675784088
642979577
392879931
483662922
234897348
602256919
650294202
121065097
391094270
448333905
1056828284
347942531
33711389
646479650
123186880
476683352
573957988
216339576
589542863
360855540
189282962
916111085
719141950
765101220
792716166
328828933
156127344
950601320
700098652
650486397
22335290
405525340
464657137
15028135
659825323
685197310
9519592
993207350
873995323
835150748