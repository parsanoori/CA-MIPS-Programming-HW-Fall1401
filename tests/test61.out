2022756
9446006
9519592
15028135
17406770
22335290
24023255
33711389
38106522
41096124
48538818
62017944
121065097
123186880
141373030
156127344
157742511
172892190
174208411
180003713
189282962
209245850
216339576
234897348
250519016
293993014
328828933
347942531
360855540
361996174
377202253
388111810
391094270
392879931
394198436
405525340
433883883
448333905
464657137
468642530
473856419
476683352
483662922
507498319
508218917
526105145
557379720
573957988
589542863
602256919
642979577
646479650
650294202
650486397
659825323
675784088
685197310
688713923
690542678
692669654
698664947
700098652
719141950
721483381
734929698
742862699
763140492
765101220
768377218
784394742
792716166
834508085
835150748
873995323
878708369
889732650
916111085
950601320
982311815
993207350
1012674018
1050300208
1056134075
1056828284